app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly make sure that `:confirmable` is also enabled in your user model, when a  user signs up the app should send a  confirmation email or prompt for user to  confirm email before they can signin successfully to the app👍🏿

Reply:
Done.